packages/itwinui-variables/src/sizes.scss

```diff
@@ -15,7 +15,7 @@
 }
 
 @mixin border-radii {
-  --iui-border-radius-1: 4px;
+  --iui-border-radius-1: var(--iui-size-2xs);
```

Comment:
I think this was intentionally left as 4px because it doesn't need to scale with base font size.

Reply:
Reverted.